lib/auth-client.js

```diff
@@ -42,16 +42,33 @@ const config = require('./config.json');
  * must exactly match one of the registered URLs.
  * @param {Boolean} [options.testMode=false] - Launch Smartcar Connect in
  * [test mode](https://smartcar.com/docs/guides/testing/).
+ * testMode is now deprecated. Use mode instead.
```

Comment:
```suggestion
 * @param {Boolean} [options.testMode=false] - Deprecated, please use `mode` instead.
 * Launch Smartcar Connect in [test mode](https://smartcar.com/docs/guides/testing/).
```
Nit: better to state that it is deprecated earlier rather than later